platforms/anchor-platform/admin-guide/sep24/integration.mdx

```diff
@@ -536,7 +536,7 @@ If a user has made a transfer, you should do a transaction recovery, and then yo
 
 ### Expired Transaction
 
-Your business may want to expire transactions that have been abandoned by the user after some time. It's a good practice to clean up inactive transactions in the `incomplete` status. To do so, simply change the transaction status to `expired`.
+Your business may want to handle abandoned transactions by expiring those have remained inactive for a certain period. To achieve this, check the transaction status using the `GET /transactions` endpoint and sort the results by the `user_action_required_by` timestamp. If the timestamp has passed, manually execute the appropriate logic, such as expiring the transaction or initiating an auto-refund, based on the transaction's current status.
```

Comment:
```suggestion
Your business may want to handle abandoned transactions by expiring those have remained inactive for a certain period. To achieve this, check the transaction status using the `GET /transactions` endpoint and sort the results by the `user_action_required_by` timestamp. If the timestamp has passed, manually execute the appropriate logic, such as expiring the transaction or initiating an auto-refund, based on the transaction's current status. 
For example, to expire transaction business should change the transaction status to `expired`:
```

Reply:
Maybe same for SEP-6 ^ just to clarify that in the example we are expiring transactions